src/github/credentials.ts

```diff
@@ -13,14 +13,17 @@ import { GitHubServer } from '../authentication/githubServer';
 import { getToken, setToken } from '../authentication/keychain';
 import { Remote } from '../common/remote';
 import Logger from '../common/logger';
+import * as PersistentState from '../common/persistentState';
 import { handler as uriHandler } from '../common/uri';
 import { createHttpLink } from 'apollo-link-http';
 import fetch from 'node-fetch';
 import { ITelemetry } from '../common/telemetry';
 
 const TRY_AGAIN = 'Try again?';
 const SIGNIN_COMMAND = 'Sign in';
+const IGNORE_COMMAND = 'Close';
```

Comment:
Let's make this "Don't show again" for clarity

Reply:
Done!